components/atoms/image/index.js

```diff
@@ -29,7 +29,7 @@ const StyledImg = styled.img`
     fluid &&
     `
     height: auto;
-    max-width: 100%;
+    width: 100%;
```

Comment:
Causes issue for messaging image icons:
![Screen Shot 2019-12-15 at 11 15 00](https://user-images.githubusercontent.com/816053/70861321-7890d080-1f2c-11ea-8702-a7b88f45ebeb.png)